react/fields/MonthPicker/MonthPicker.test.js

```diff
@@ -0,0 +1,175 @@
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import React from 'react';
+import {
+  createRenderer
+} from 'react-addons-test-utils';
+import { findAllWithClass } from 'react-shallow-testutils';
+import MonthPicker from './MonthPicker';
+
+chai.use(sinonChai);
+
+const renderer = createRenderer();
+
+describe('MonthPicker', () => {
```

Comment:
🤔  `MonthPicker` simply passed through a `value` & `onChange` function. All we can really test is that it forwards those props. I've done that for onChange & onBlur so I can add one for value as well.